app/scaffold-controllers/searchBarController.js

```diff
@@ -0,0 +1,97 @@
+define([
+    'bluebird',
+    'plugins/webViewPlugin'
+],
+function(
+    Promise,
+    WebViewPlugin
+) {
+    var SearchBarController = function(webView, layout, searchConfig) {
+        this.viewPlugin = webView;
+        this.layout = layout;
+        this.searchConfig = searchConfig;
+
+        // Track the state to show.hide keyboard on toggle
+        //
+        // Set to true because it will be hidden when added
+        // to the parent layout
+        this.isVisible = true;
```

Comment:
I added the `Set to true` comment because I didn't initially understand why we would want it set to true when we are adding it. I will update the comment to make it more clear. I think there is value in having the comment.